spec/unit/typesafe/config/concatenation_spec.rb

```diff
@@ -45,4 +45,383 @@
     expect(e.message).to include("null")
     expect(e.message).to include('{"x":"y"}')
   end
+
+  it "no arrays in string concat" do
```

Comment:
yeah, fair point.  Right now, these strings are all direct translations of the `@Test` method names in the java version, and it's actually been extremely useful to have that super-obvious mapping between the two languages.  Yet another instance where I'm favoring consistency over being idiomatic.  This poor project was just doomed to be non-idiomatic from the start.